pages/AnimationPost/[index].js

```diff
@@ -85,9 +85,9 @@ const AnimationPost = props => {
   );
 };
 
-AnimationPost.getInitialProps = ({ query }) => {
+AnimationPost.getInitialProps = ({query}) => {
```

Comment:
{query} -> { query } because it's destructure

Reply:
updated.